python-lib/dku_stress_test_center/stress_test_center.py

```diff
@@ -54,7 +55,7 @@ def check_relevance(self, preprocessing):
                 self.not_relevant_explanation = "The scaling factor is set to 1."
         elif type(self.shift) is MissingValues:
             for feature in self.features:
-                if preprocessing[feature]["missing_handling"] == "DROP_ROW":
+                if preprocessing[feature].get("missing_handling", "") == "DROP_ROW":
```

Comment:
I think this is enough :)
```suggestion
                if preprocessing[feature].get("missing_handling") == "DROP_ROW":
```